content/Watsonx/WatsonxAI/files/rag-with-discovery.ipynb

```diff
@@ -263,11 +254,17 @@
     "########################################################################################\n",
     "import re\n",
     "\n",
-    "augmented_prompt = augment( prompt_template, combined_disc_results, discovery_question )\n",
-    "output = generate( model, augmented_prompt )\n",
-    "if not re.match( r\"\\S+\", output ):\n",
-    "    print( \"The model failed to generate an answer\")\n",
-    "print( \"\\nAnswer:\\n\" + output )\n"
+    "def query_watsonx(question):\n",
+    "    augmented_prompt = augment( prompt_template, combined_disc_results, question )\n",
```

Comment:
Looks like a Python error here: `NameError: name 'combined_disc_results' is not defined`

Reply:
Just when I run the last cell.

<img width="827" alt="Screenshot 2023-10-16 at 17 41 13" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/6494561/2407c4aa-5373-4093-b543-93b4aa56c907">